textbook/11/1/Rules_Definitions.ipynb

```diff
@@ -18,7 +18,7 @@
     "\n",
     "**Complement of an event**: The complement of an event $A$ is the event that $A$ does not occur, denoted by $A^C$. For the event $A$ defined above, $A^C$ is the event that more than half of the babies are boys, or the set of integers from 50 to 100.\n",
     "\n",
-    "![](./img/venn_comp.png)\n",
+    "<img src=\"./img/complement.png\" width=\"400\"/> \n",
```

Comment:
Any reason to use `<img>` and to set a width?

* As far as I can tell, one of the advantages to using the Markdown-native syntax is that the textbook builder detects what you're doing and ensures the image is included. With this version, none of the images are included and these tags are broken. For that reason, I think we don't really want to use `<img>` tags at all.
* The width is probably fine, but generally it's nice to allow the browser to figure out the best sizing, based on the image and layout. (And if we do need to control the sizing, there might be a more dynamic way.)

I would just tweak these to work with the textbook – the Markdown syntax is both textbook-compatible and less verbose – but, let me know if there's a reason behind the particular form you used.

![image](https://user-images.githubusercontent.com/530998/200374685-d70ba13b-80bc-4aa7-bfa4-f919f39ade06.png)